extension/module/module.h

```diff
@@ -63,6 +63,8 @@ class Module {
   explicit Module(
       const std::string& file_path,
       const LoadMode load_mode = LoadMode::File,
+      std::unique_ptr<runtime::MemoryAllocator> memory_allocator = nullptr,
```

Comment:
nit: method_allocator

Reply:
oh I see. This is supposed to be called method_allocator?